### DISCUSSION TITLE
More custom stuff to motivate people to support

### DISCUSSION THREAD

**Discussion by soupychemist** [ppy/osu#26255] [Ideas]
I feel like more silly people (like me) would support osu more if it had the following features for custom profiles.
-Custom profile background color 
-A font or maybe a font color so people can match with background color and even the existing image display that's there. 
-Titles (if anyone's played val or fall guys kinda like that, just silly little titles related to osu or skill in osu could even be based off medals earned)

I paid mainly because I wanted to display my fav character and so did 2 of my friends. Customization can't be underestimated because so many people will pay just for that sort of like nitro on discord its mainly to make your profile pop nowadays.

**Comment by Rekunan** [comment]
Supporter is meant to be something you pay for out of support, not to achieve extra features.

However, I do agree with adding these features, and they do make sense to lock behind supporter considering extra server costs.
